src/Chapter2.hs

```diff
@@ -621,7 +647,15 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven = go
+  where
+    go [] = []
+    -- Wrong even for positions not element ...
+    -- go (x:xs) = if even x then x : go xs else go xs
```

Comment:
Yes, this would work for the task of having only even numbers 🙂 But in that case, you could also use `filter` function 😉

Reply:
Ok, I will keep `filter` in mind. 😃